code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -481,10 +481,11 @@
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/mob/living/silicon/pai/proc/softwareMedicalRecord() called tick#: [world.time]")
 	var/dat = ""
 	if(src.subscreen == 0)
+		dat += "<a href='byond://?src=\ref[src];software=medicalsupplement;sub=2'>Host Bioscan</a><br>"

```

Comment:
[14:47:51] [14:47:51] <Ririchiyo> in software.dm at line 484
[14:48:54] [14:48:54] <Ririchiyo> do we have links with byond://?